library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -405,6 +424,33 @@ public int getSectionPosition(String tag) {
     }
 
     /**
+     * Return the section position in the adapter.
+     *
+     * @param section This section
+     * @return position of the section in the adapter
+     */
+    public int getSectionPosition(Section section) {
```

Comment:
You will have to create a new branch from latest develop branch, then you can [cherry-pick](https://git-scm.com/docs/git-cherry-pick) specific commits to this new branch.
If you don't have a separate commit for this method then unfortunately you will have to make the changes again manually.